kogito-timer-springboot/README.md

```diff
@@ -15,11 +15,66 @@ This example shows
 * optionally use Job Service that allows to externalize time tracking to separate service and by that offload the runtime service
 	
 	
-Intermediate timer event
-<p align="center"><img width=75% height=50% src="docs/images/process.png"></p>
+* Intermediate timer event
+<p align="center"><img width=75% height=50% src="docs/images/timers.png"></p>
+
+* Intermediate timer Diagram Properties
+<p align="center"><img src="docs/images/timersDiagramProperties.png"></p>
+
+* Intermediate timer Diagram Properties
+<p align="center"><img src="docs/images/timersDiagramProperties2.png"></p>
+
+* Intermediate timer Before Timer
+<p align="center"><img src="docs/images/timersBeforeTimerScriptTask.png"></p>
+
+* Intermediate timer Timer
+<p align="center"><img src="docs/images/timersTimer.png"></p>
+
+* Intermediate timer After Timer 
+<p align="center"><img src="docs/images/timersAfterTimerScriptCall.png"></p>
+
+* Boundary timer event
+<p align="center"><img width=75% height=50% src="docs/images/timerOnTask.png"></p>
+
+* Boundary timer Diagram Properties 
+<p align="center"><img src="docs/images/timerOnTaskDiagramProperties.png"></p>
```

Comment:
maybe change the name property in screenshot/bpmn file to `timersOnTask`

Reply:
fixed @radtriste